service/matching/matching_engine_test.go

```diff
@@ -170,6 +170,8 @@ func (s *matchingEngineSuite) SetupTest() {
 	s.mockMatchingClient.EXPECT().ReplicateTaskQueueUserData(gomock.Any(), gomock.Any()).
 		Return(&matchingservice.ReplicateTaskQueueUserDataResponse{}, nil).AnyTimes()
 	s.taskManager = newTestTaskManager(s.logger)
+	s.taskManager.dbConditionalFailedError = false // set to true when you want db to throw ConditionalFailedErrors randomly
+	s.taskManager.dbServiceError = false           // set to true when you want db to throw ServiceErrors
```

Comment:
false is the zero value so they should be false already.. these lines have no effect, do they?